app/models/document.rb

```diff
@@ -106,6 +106,10 @@ def first_published_date
     published_edition.first_public_at if published?
   end
 
+  def first_published_on_govuk
+    edition_versions.where(state: "published").first.try(:created_at)
```

Comment:
Nit: probably nicer to just pluck the date here instead of getting the model and discarding it. 

There was a new(ish) method added, pick, that'd do a good job: https://sikac.hu/relation-pick-in-rails-6-0-87b2112f6097